README.md

```diff
@@ -72,94 +72,111 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.61 - July 2022 Update
+### 0.62 - August 2022 Update
 
 This is a lighter release, with a shorter development cycle and focused on stability and improvements.
 
 **Highlights**
```

Comment:
Maybe we could add the gifs for the new utilities in the changelog. (Not in the readme though),

Reply:
I don't think we need to. Changelogs are usually text. There'll be OOBE and online docs and such...

Reply:
This isn’t a reason to block the PR